using_openshift/builds.adoc

```diff
@@ -17,7 +17,7 @@ We defined three types of builds:
 * STI build
 * custom build.
 
-More of this is described link:../architecture/builds.html[here].
+These are described in more detail link:../architecture/builds.html[here].
```

Comment:
Just a note, when linking to other topic files, I've lately been using a form like the following, explicitly referring to the topic title in the sentence:

~~~
These are described in more detail in the link:../architecture/builds.html[Builds] topic.
~~~

Versus using a generic "here". I don't know that we've settled on a style for this kind of thing yet, but I just wanted to bring it up, and we can follow up offline with a discussion with @CowboysFan.